src/prisma/schema.prisma

```diff
@@ -1,6 +1,6 @@
 datasource db {
     provider = "sqlite"
-    url      = "file:./data/bot.db"
+    url      = env("DATABASE_URL")
```

Comment:
There is no documentation on README on how to setup this on `docker-compose.yml`.

Possibly
```yml
build:
  context: .
  args:
    DATABASE_URL: "file:/app/data/bot.db"
```